Frontend/ui-common/src/Common/Constants/Constant.purs

```diff
@@ -0,0 +1,37 @@
+
+module Common.Constants.Constant where
+
+import Common.Styles.Colors (shadowGray)
+
+type AndroidPolylineAnimationConfig = {
+    color :: String
+  , drawDuration :: Int 
+  , fadeDuration :: Int
+  , delayDuration :: Int
+}
+
+type IOSPolylineAnimationConfig = {
```

Comment:
try to unify the types as both are having same type and meaning.

Reply:
for IOS we want to pass percentage and for android we need duration. unifying will cause less meaningful.